templates/crawler/cookiecutter.json

```diff
@@ -0,0 +1,12 @@
+{
+    "project_name": "crawlee-python-beautifulsoup-project",
```

Comment:
Is it correct?

Reply:
Good point. I'll probably just change it to my-crawler or something.